client/src/components/Sidebar/SidebarBody/index.tsx

```diff
@@ -0,0 +1,15 @@
+import React from 'react'
+import { SIDEBAR_ITEM } from '../'
+import UserList from '../UserList'
+import ChatHistory from '../ChatHistory'
+
+const SidebarBody = ({ currentItem }: { currentItem: SIDEBAR_ITEM }) => {
+  return (
+    <div className="sidebar--body">
```

Comment:
I'm seeing this (the dash dash) in multiple places. Is this intentional?

Reply:
Trying to do `element` part, but it's quickly becoming a mess. (in this specific case, it should actually be `sidebar__body` instead of `sidebar--body`
https://sass-lang.com/styleguide/code